### PR TITLE
fix ESLint warn: spectrum/api/migrations/config.js

### PR DESCRIPTION
**Pull request by ryota-murakami**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

## overview

**cmd:**
node_modules/.bin/eslint api/migrations/config.js

**warn:**
spectrum/api/migrations/config.js
  16:3  warning  Unreachable code  no-unreachable

thank you for your review😀

### CODE REVIEW

**Comment by ryota-murakami** [comment]
ping👉

File: api/migrations/config.js [hunk 1]
```diff
@@ -13,10 +14,9 @@ const RUN_IN_PROD = !!process.env.AWS_RETHINKDB_PASSWORD;
 
 if (RUN_IN_PROD && process.argv[4] === 'down') {
   throw new Error('Do not drop the production database!!!!!');
-  process.exit(1);
```
Comment (brianlovin) [inline api/migrations/config.js:16]: Why is this line being removed?

Comment (ryota-murakami) [reply api/migrations/config.js:16]: Eslint says this is unreachable code. https://eslint.org/docs/rules/no-unreachable

Comment (brianlovin) [reply api/migrations/config.js:16]: Ah, fair enough!

**Review by brianlovin — APPROVED** [review]

**Pull request merged**